portfolio/src/main/webapp/index.html

```diff
@@ -45,7 +45,14 @@ <h2>LinkedIn and Github</h2>
             <p>Click here to get a random fact:</p>
             <button onclick="addRandomFact()">Press</button>
             <div id="fact-container"></div>
+            <p>Click here to get a message from a servlet.</p>
+            <button onclick="fetchMessage()">Click</button>
+            <div id="message-container"></div>
+            <form action="/data" method="POST">
+                <textarea name="comment-input">Enter a comment</textarea>
+                <input type="submit"/>
+            </form>
```

Comment:
Let's separate the form from the elements above it with a new line to make it easier to read. Please do the the same for the section above it which contains of a p, button and div for the "fetchMessage()".

Reply:
I have updated the layout.